### DISCUSSION TITLE
How to toggle ºC/ºF?

### DISCUSSION THREAD

**Discussion by Ctrl-LeftAlt-Delete** [devbis/z03mmc#86] [Q&A] [answered]
I saw v1.0.6 added the ability to toggle ºC/ºF but I can't figure out how. The device is showing on my Home Assistant. When I go to Manage Zigbee Device I can see a bunch of clusters including TemperatureMeasurement but I can't find any attribute to change units.
Thanks!

**Comment by devbis** [comment, marked as answer]
Is it a duplicate of https://github.com/devbis/z03mmc/issues/78 ?

**Reply by Ctrl-LeftAlt-Delete** [reply]
Yes :P apologies for the dupe.

**Comment by devbis** [comment]
Added toggling units by pressing for 1 sec reset-gnd in version 1.1.0
